pkg/provider/provider.go

```diff
@@ -43,7 +43,7 @@ import (
 )
 
 const (
-	AffinityAllowedKey               = "AffinityAllowed"
+	AffinityRequiredKey              = "AffinityRequired"
```

Comment:
Add a note in the Readme about the new "AffinityRequired" label

Reply:
Done.